### DISCUSSION TITLE
change path from binlog?

### DISCUSSION THREAD

**Discussion by simonmau** [egramtel/tdsharp#188] [Q&A]
Hi! 
Is it possible to change the default path from the td.binlog file?
I would love to run this in a docker container and map a directory containing the file - so i can recreate the container without the need to login again.

**Comment by ForNeVeR** [comment]
I think you can achieve this by calling `SetTdLibParameters`, [see the original reference](https://core.telegram.org/tdlib/docs/classtd_1_1td__api_1_1tdlib_parameters.html) (`database_directory_`).
